modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -241,7 +242,7 @@ def fetch_clinic_appointments(start_time, end_time, statuses)
         get_appointments(start_time, end_time, statuses)[:data].select { |appt| appt.kind == 'clinic' }
       end
 
-      def prepare_appointment(appointment)
+      def prepare_appointment(appointment, include = {})
```

Comment:
FYI @cferris32 I expect this to conflict with your AVS PR so I'm going to hold this back until you're done.